sdk/storage/azure-storage-file-share/azure/storage/fileshare/_shared_access_signature.py

```diff
@@ -137,7 +137,7 @@ def generate_share(self, share_name, permission=None, expiry=None,
         :param ShareSasPermissions permission:
             The permissions associated with the shared access signature. The
             user is restricted to operations allowed by the permissions.
-            Permissions must be ordered read, create, write, delete, list.
+            Permissions must be ordered read, write, delete, list.
```

Comment:
Is there a reason we want to delete create here? https://docs.microsoft.com/en-us/rest/api/storageservices/create-service-sas#permission-for-a-share

Reply:
I based it off of the class: https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/storage/azure-storage-file-share/azure/storage/fileshare/_models.py#L725

Which has read, write, delete, list.

Are you saying the class is incorrect?

Reply:
I based it off of the class: https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/storage/azure-storage-file-share/azure/storage/fileshare/_models.py#L725

Which has read, write, delete, list.

Are you saying the class is incorrect?

Reply:
I think the class is not accurate.. The rest doc I pasted there should be accurate

I created a issue(Rakshith linked the issue in the conversation) which is to make every class accurate according to rest doc

Reply:
Can you link to that issue here?  Thanks

Reply:
Done 😊

Reply:
For clarity, pasting the link actually in this conversation:
https://github.com/Azure/azure-sdk-for-python/issues/8799